Update reset mfa (#187)
GovTechSG/techpass-user-docs: docs/reset-mfa.md

# Reset MFA

TechPass users need to reset MFA when they do any of the following:

- Change their mobile phone.
- Delete Authenticator app from their mobile phone.

> - Get a new TechPass account.

Reviewer: it's not reset mfa for new accounts right? it's setup actually. so don't think the last option is applicable

Author: If a vendor who has changed organisation (say from cognizant to techmahindra) , he will get a new techpass account and in that case should he follow reset TechPass (because he may still have the old profile on his authenticator or is that out od scope for reset mfa) That was my intention while adding the third use case

similarly, what if a officer changes agencies and gets a new WOG account? Should that also be out od scopr of this doc?

Author: noted. thanks.

Users whose TechPass domain is ```techpass.gov.sg```,  will only have TechPass profile(account) and must [Reset TechPass MFA](reset-techpass-mfa-for-new-device) when needed.

> Users whose TechPass domain is not ```techpass.gov.sg``` will have two profiles(accounts) - WOG and TechPass. Such users must do the following:

Reviewer: ```suggestion
Users whose TechPass domain is not ```techpass.gov.sg``` will have two profiles(accounts) - SGGovt M365(WOG) and TechPass. Such users must do the following:
```

> - [Reset MFA for their WOG profile](reset-security-verification-for-wog-account) only if you have changed the mobile phone or deleted the Authenticator app from your mobile phone.

Reviewer: ```suggestion
- [Reset WOG MFA profile](reset-security-verification-for-wog-account) only if you have changed the mobile phone or deleted the Authenticator app from your mobile phone.
```

Author: agreed, but will it be better if I replace all **WOG** with **SGGovt M365** ?

Author: noted, thanks!

# Reset multifactor authentication (MFA)

TechPass users need to reset MFA if they:

- Change their mobile phone.
- Delete Authenticator app from their mobile phone.

Users whose TechPass domain is ```techpass.gov.sg```,  will only have TechPass profile(account) and must [Reset TechPass MFA](reset-techpass-mfa-for-new-device) when needed.

Users whose TechPass domain is not ```techpass.gov.sg``` will have two profiles(accounts) - SGGovt M365(WOG) and TechPass. Such users need do the following:

- [Reset WOG(SGGovt M365) MFA](reset-security-verification-for-wog-account).

- [Create service request](https://go.gov.sg/techpass-sr) to remove the MFA of their TechPass profile.

> - When the request is processed successfully, the support team sends an email with instructions on how to reset MFA for the TechPass account.

Reviewer: ```suggestion
- When the request is processed successfully, you will be able to register a new TechPass MFA profile when you try to sign in again.
```

Author: hmm, the reason i have phrased it in such a manner because 
1. This is what is happening
2. The email tells what to do next .

If I says you will be able to register a new TechPass MFA profile, user will ask then how to do it? For that, if I link user to https://docs.developer.tech.gov.sg/docs/techpass-user-guide/onboard-public-officers-using-non-se-machines?id=step-4-onboard-to-techpass

it does not have the correct starting point. This was written from the onboarding part.  (we need to revisit if step 4 in PO onbaording has to match with the steps in the email)

 I felt the one on the email has it clearly.
